app/controllers/application_controller.rb

```diff
@@ -630,6 +630,22 @@ def onboarding_topbar_props
 
   helper_method :onboarding_topbar_props
 
+  def topbar_props
+    {
+      logo: {
+        href: '/',
+        text: @current_community.name(I18n.locale),
+        image: @current_community.wide_logo.present? ? @current_community.wide_logo.url(:header) : nil,
+        image_highres: @current_community.wide_logo.present? ? @current_community.wide_logo.url(:header_highres) : nil
+      },
+      search_mode: 'keyword-and-location',
+      search_keyword_placeholder: 'Search...',
+      search_location_placeholder: 'Location'
```

Comment:
This _might_ be out of scope for this PR, but we probably should agree about a format for subcomponent props. We're using two different ones here: `logo` subobject vs separate search params. I see clear benefits in the subobject format: The container component doesn't need to care about the props of it's children, and Proptypes don't need to be written in two places.